ci_jobs_trigger/libs/operators_iib_trigger/iib_trigger.py

```diff
@@ -108,9 +108,7 @@ def get_new_iib(config_data, logger):
     new_trigger_data = False
     data_from_file = get_iib_data_from_file(config_data=config_data)
     new_data = copy.deepcopy(data_from_file)
-    if (ci_jobs := config_data.get("ci_jobs", {})) is None:
```

Comment:
why is this removed? no point in continuing if there are not ci jobs

Reply:
moved this check to fetch_update_iib_and_trigger_jobs func, it will be called before get_new_iib() and get_jobs_name_from_config()